tests/features/EditorRevisionPermissions.feature

```diff
@@ -0,0 +1,34 @@
+@api
+Feature: Editor Revision Permission Settings
+  As a user with the Editor role
+  I am able to view revision settings when adding new content if I have permission to view, revert or delete revisions
+  And if I don't have those permissions
+  I am unable to view the revision settings when adding new content
+  So I only see the settings that are available to me
+
+
+Scenario: Verifying that the Editor role has the correct permissions
+  Given I am logged in as a user with the "administrator" role
+   When I visit "/admin/people/permissions"
+    And I check the box "edit-4-view-revisions"
+    And I check the box "edit-4-revert-revisions"
+    And I check the box "edit-4-delete-revisions"
+    Then I press "Save permissions"
+
+Scenario: As an Editor, I should see the revision information section when adding new content because I have the correct permissions
```

Comment:
Combine the first two scenarios and the last two scenarios into just 2 scenarios. Even though you have to log in as two different accounts during the scenario, it's all connected.